src/bfx_tools/optitype.ml

```diff
@@ -41,6 +41,7 @@ let hla_type ~work_dir ~run_with ~r1 ?r2 ~run_name nt =
     in
     object
       method is_done = Some (`Volume_exists vol)
+      method path = work_dir
```

Comment:
You put the path to the directory because optitype creates new filenames everytime right?